Simplenote/src/main/java/com/automattic/simplenote/DeepLinkActivity.kt

```diff
@@ -88,6 +93,21 @@ class DeepLinkActivity : AppCompatActivity() {
         }
     }
 
+    private fun handlePasswordReset(uri: Uri) {
+        val redirectParam = uri.getQueryParameter("redirect")
+        if (redirectParam == "simplenote://launch") {
```

Comment:
Question (❓): The logic within both `if/else` seems the same to me, what that done on purpose? 🤔

Reply:
oof good catch, fixed in 2589709f - it should just launch the app when capturing a password reset link and the app is installed (this to do the same thing it was doing before, no behavior change)